test/fixtures/test-runner/dependent.js

```diff
@@ -1 +1,2 @@
 require('./dependency.js');
+import('./dependency.mjs');
```

Comment:
Let's try to add a dependency that's not a file.
```suggestion
import('./dependency.mjs');
import('data:text/javascript,');

```